examples/OpenGLWindow/Win32Window.cpp

```diff
@@ -446,9 +446,13 @@ void Win32Window::setWindowTitle(const char* titleChar)
 		SetWindowTextA(m_data->m_hWnd, titleChar);
 #else
 		DWORD dwResult;
+		int wideCharCount = MultiByteToWideChar(CP_UTF8, 0, titleChar, -1, NULL, 0);
+		wchar_t *wideTitleString = new wchar_t[wideCharCount];
+		MultiByteToWideChar(CP_UTF8, 0, titleChar, -1, wideTitleString, wideCharCount);
 		SendMessageTimeout(m_data->m_hWnd, WM_SETTEXT, 0,
```

Comment:
This problem arises when you are creating a unicode build.
Does replacing `SendMessageTimeout` with `SendMessageTimeoutA` also fix your problem?